command_list.md

```diff
@@ -0,0 +1,116 @@
+# Account
+```
+rucio account list
+rucio account add
+rucio account show
+rucio account remove
+rucio account history
+rucio account update
+rucio account ban add
+rucio account ban remove
+rucio account attribute list
+rucio account attribute add
+rucio account attribute remove
+rucio account limit list
+rucio account limit add
+rucio account limit remove
+rucio account identity list
+rucio account identity add
+rucio account identity remove
+```
+
+# Config 
+```
+rucio config list
+rucio config add
+rucio config remove
+```
+
+# DID 
+```
+rucio did list
+rucio did show
+rucio did remove
+rucio did touch
+rucio did parent
+rucio did attach
+rucio did detach
+rucio did upload
+rucio did download
+rucio did content list
+rucio did content history
+rucio did content close
+rucio did content open
+rucio did metadata list
+rucio did metadata add
+rucio did metadata remove
+rucio did container add
+rucio did dataset add
```

Comment:
I don’t think it’s worth having the `container` and `dataset` groups. Both of them are collections, and you can’t add a file DID. I would recommend putting the type of collections as an option to `did add`.